docs/Developers/Git_Basics.rst

```diff
@@ -44,15 +44,25 @@ To do this, you can use these to clone the coala/coala-bears repositories:
 
 ::
 
-    $ git clone https://github.com/coala/coala
+    $ git clone -o upstream https://github.com/coala/coala
 
 or
 
 ::
 
-    $ git clone https://github.com/coala/coala-bears
+    $ git clone -o upstream https://github.com/coala/coala-bears
+
+.. note::
+
+    ``-o upstream`` sets the remote name of the original coala/coala-bears
```

Comment:
>Look at the next line. There's a line wrap for length reasons.

@PierceAndy The change requested by supergr35 was to change "coala/coala-bears repository"  to "coala and coala-bears repositories". 
You've changed "repository" to "repositories" in L58 but not changed "coala/coala-bears" to "coala and coala-bears" in L57. 
I'm sorry if there was some confusion previously :sweat_smile: